src/server/game/Movement/MovementGenerators/HomeMovementGenerator.h

```diff
@@ -21,18 +21,15 @@
 
 #include "MovementGenerator.h"
 
-class Creature;
```

Comment:
I guess someone thought it was needed to declare the class for the template defines or w/e, but the thing is its not needed. Wether its because the MovementGenerator.h include or the class inherit I dont care, just checked its not needed.